enhancements/update/update-blocker-lifecycle/README.md

```diff
@@ -0,0 +1,109 @@
+---
+title: update-blocker-lifecycle
+authors:
+  - "@wking"
+reviewers:
+  - "@LalatenduMohanty"
+approvers:
+  - TBD
+creation-date: 2020-09-11
+last-updated: 2021-03-30
+status: implementable
+---
+
+# Update-blocker Lifecycle
+
+We occasionally have bugs which impact update success or the stability of the target release.
+When that happens, we protect users by [removing the impacted updates from our update recommendations][graph-data-block].
+This enhancement describes the process used to identify these bugs and determine which edges should be blocked because of them.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
```

Comment:
Nothing in the summary of this proposal addresses the core/root issue (the underlying product quality issue that is leading to the us having to 'block' edges in the first place. 

All this proposal does; is enhance our ability for individuals to 'signal' things that 'might' be a possible issue; and for decision makers to 'find' those issues in the vast sea of issues that we face day to day.